src/compiler/objective_c_plugin.cc

```diff
@@ -87,7 +87,11 @@ class ObjectiveCGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
          ++param_str) {
       std::vector<::grpc::string> param =
           grpc_generator::tokenize(*param_str, "=");
-      if (param[0] == "generate_for_named_framework") {
+      cerr << param[0];
+      if (param.size() != 2) {
+        *error = grpc::string("Input format: <key1>=<value1>,<key2>=<value2>");
```

Comment:
Some parameters need to allow key-only, without value. I think this check should be in the `if` statement of `generate_for_named_framework` itself, like `generate_for_named_framework parameter requires a value.`